format_code.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
```

Comment:
This won't work on macOS; does that matter?

Reply:
I don't have Apple hardware to test on, but I would like to this to work there if possible. I'll try to find an alternative. (Do you happen to know what it should be?)

Reply:
I've previously used [this approach](https://github.com/lcm-proj/lcm/blob/master/lcm-java/lcm-spy.sh.in#L4-L7)... which seems appropriate, since LCM is already using it :smile:.

Reply:
Works for me, thank you.